container/target/target.tf

```diff
@@ -38,7 +38,7 @@ locals {
   second_split    = "${split("/", element(local.splits, 1))}"
   health_protocol = "${element(local.splits, 0)}"
   health_port     = "${element(local.second_split, 0)}"
-  health_path     = "${local.health_protocol != "TCP" ? "/${element(local.second_split, 1)}" : ""}"
+  health_path     = "/${join("/", slice(local.second_split, 1, length(local.second_split)))}"
```

Comment:
Are we sure that `to` in `slice(list, from, to)` does not need to be `length(list) - 1` since the list is zero indexed? Could have some surprising behaviour if we are actually out of bounds here?

Reply:
I tried length(list) -1 first and it missed off the last element